core/src/main/java/com/linecorp/armeria/common/ContentTooLargeException.java

```diff
@@ -23,20 +23,65 @@
 
     private static final long serialVersionUID = 4901614315474105954L;
 
+    private long transferred;
+    private long total;
+    private long maximum;
+
     private static final ContentTooLargeException INSTANCE = new ContentTooLargeException(false);
 
     /**
      * Returns a {@link ContentTooLargeException} which may be a singleton or a new instance, depending on
      * {@link Flags#verboseExceptionSampler()}'s decision.
+     * @deprecated use {@link #builder()} instead.
      */
+    @Deprecated
```

Comment:
I guess we don't really need to deprecate this method. It's still gonna be useful when creating an exception quickly, mostly for testing.